cheatsheets/Drone_security_sheet.md

```diff
@@ -0,0 +1,173 @@
+# Drone Security Cheat Sheet
+
+## Introduction
+
+Drone security is crucial due to their widespread adoption in industries such as military, construction, and community services. With the increasing use of drone swarms, even minor security lapses can lead to significant risks.
+
+This cheat sheet provides an overview of vulnerable endpoints in drone systems and strategies to mitigate security threats.
+
+---
+
+## Drone System Components
+
+A typical drone architecture consists of three main components:
+
+1. **Unmanned Aircraft (UmA)** – The physical drone itself, including its sensors and onboard systems.
+2. **Ground Control Station (GCS)** – The interface used to control and monitor drone operations.
+3. **Communication Data-Link (CDL)** – The network connection between the drone and the GCS
+
+The communication between the drone and the GCS is vulnerable to interception and attacks. This will be made evident in the future sections as well. It is important to understand that peripherals attached to drone may be vulnerable too! To explain this, we have made a list of **vulnerable endpoints** below.
+
+---
+
+## Vulnerable Endpoints & Security Risks
+
+### 1. Communication Security
+
+- **Insecure Communication Links** – Data transmitted between the drone and GCS can be intercepted if not properly encrypted. Use standard protocols for encryption of any data being sent over.
+
+- **Spoofing and Replay Attacks** – If the drone uses a GPS module then data spoofing and command replay attacks can also become a reality. Again encrpyted data transfer is the best way to go forward. There are many more methods, which have been discussed [here](https://www.okta.com/identity-101/gps-spoofing/)
+
+- **Wi-Fi Weaknesses** – Weak authentication or unprotected channels can allow unauthorized access. This is even possible through simple [microcontrollers like ESP8266](https://github.com/SpacehuhnTech/esp8266_deauther)!
+
+    - Use **802.11w MFP (Management Frame Protection)** to prevent Wi-Fi deauthentication attacks. Don't worry, if your Wi-Fi systems are up to date, then this is a default protocol now.
+
+### 2. Authentication & Access Control
+
+Most drone controllers use 2 sets of computers,
+
+1. The main chip that performs the PID control and handles motors
+
+2. An additional SoC (called the **companion computer**) to manage peripherals (like the cameras, LiDARs etc.) and send telemetry data.
+
+Thus, it becomes very important to maintain their security as well. The possible risks in this case are:
+
+- **Companion Computers** – Open ports (e.g., SSH, FTP) can be exploited if not securely configured.
+
+- **User Error and Misconfiguration** – Misconfigured security settings can expose the drone to risks.
+
+### 3. Physical Security
+
+If you're done is ever captured or lost, you should ensure that its not physically possible to steal data from it. This may happen under the following conditions:
```

Comment:
```suggestion
If your drone is ever captured or lost, you should ensure that its not physically possible to steal data from it. This may happen under the following conditions:
```